### PR TITLE
Heat map testing, slight heatmap plate refactor

### PR DESCRIPTION
**Pull request by luciipak**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@curi-bio/mantarray-frontend-components",
-  "version": "0.3.0",
+  "version": "0.4.0",
```
Comment (luciipak) [inline package.json:3]: this was from my last pull, do I need to add another? I wasn't sure with just testing updates.

File: components/basic_widgets/StimulationStudioPlateWell.vue [hunk 2]
```diff
@@ -5,6 +5,7 @@
       :style="'top:' + computed_top + 'px;' + 'left:' + computed_left + 'px;'"
     >
       <PlateWell
+        class="well"
```
Comment (luciipak) [inline components/basic_widgets/StimulationStudioPlateWell.vue:8]: for testing

File: components/plate_based_widgets/mapeditor/PlateHeatMap.vue [hunk 3]
```diff
@@ -52,6 +52,7 @@
       height="103"
     >
       <PlateWell
+        :id="'well_' + well_index"
```
Comment (luciipak) [inline components/plate_based_widgets/mapeditor/PlateHeatMap.vue:55]: for testing

File: store/modules/heatmap/getters.js [hunk 4]
```diff
@@ -1,45 +1,45 @@
...
-  heatmap_options_on_idx(state) {
-    return state.heatmap_options_array[state.heatmap_options_idx];
-  },
   heatmap_options_gradient(state) {
     return state.heatmap_options_gradient;
   },
-  heatmap_options_on_gradient(state) {
-    return state.heatmap_options_gradient[state.heatmap_options_array[state.heatmap_options_idx]];
-  },
+  // heatmap_idx(state) {
```
Comment (luciipak) [inline store/modules/heatmap/getters.js:12]: I commented out a lot of these for now, they are not currently being used.

File: tests/unit/components/basic_widgets/RadioButtonWidget.spec.js [hunk 5]
```diff
@@ -76,22 +76,4 @@ describe("RadioButtonWidget.vue", () => {
       ],
     ]);
   });
-  test("Given that there is a pre-selected radio button, When the RadioButtonWidget is mounted, Then the state of the pre-selected radio button is `selected`", async () => {
```
Comment (luciipak) [inline tests/unit/components/basic_widgets/RadioButtonWidget.spec.js:79]: this was a failing test

**Comment by jperson** [comment]
It looks like a change in this PR broke a required y-axis zoom test, not sure why the .png files didn't get added to the PR when the tests ran, but you can see the difference here

Base:
y-axis-controls-settings-popover-message-base
Actual:
y-axis-controls-settings-popover-message-actual
Diff:
y-axis-controls-settings-popover-message-actual-diff

Something with the radio button

**Comment by luciipak** [comment]
> It looks like a change in this PR broke a required y-axis zoom test, not sure why the .png files didn't get added to the PR when the tests ran, but you can see the difference here
> 
> Base:
> y-axis-controls-settings-popover-message-base
> Actual:
> y-axis-controls-settings-popover-message-actual
> Diff:
> y-axis-controls-settings-popover-message-actual-diff
> 
> Something with the radio button

Okay I reverted the change I'd made, do hopefully works now, I'll look at the preselected again later. Where was that test?

**Review by jperson — APPROVED** [review]

**Pull request merged**